packages/remix-dev/cli/setup.ts

```diff
@@ -74,9 +82,9 @@ export async function setupRemix(platform: SetupPlatform): Promise<void> {
     ".js"
   );
 
-  await fse.writeFile(path.join(remixPkgDir, "index.d.ts"), magicTypes);
-  await fse.writeFile(path.join(remixPkgDir, "index.js"), magicCJS);
-  await fse.writeFile(path.join(remixPkgDir, "esm/index.js"), magicESM);
+  await fse.writeFile(path.join(remixPkgDir, "dist/index.d.ts"), magicTypes);
+  await fse.writeFile(path.join(remixPkgDir, "dist/index.js"), magicCJS);
+  await fse.writeFile(path.join(remixPkgDir, "dist/esm/index.js"), magicESM);
```

Comment:
This will probably make Windows happy again

```suggestion
  await fse.writeFile(path.join(remixPkgDir, "dist", "index.d.ts"), magicTypes);
  await fse.writeFile(path.join(remixPkgDir, "dist", "index.js"), magicCJS);
  await fse.writeFile(path.join(remixPkgDir, "dist", "esm", "index.js"), magicESM);
```